docs/requirements.txt

```diff
@@ -19,7 +19,7 @@ mistune==2.0.3
 packaging==23.0
 pluggy==0.13.1
 pydata-sphinx-theme==0.7.2
-Pygments==2.14.0
+Pygments==2.15.1
```

Comment:
supposed to be in this PR or fetched accidentally?

Reply:
yeah, I thought we could make the update while at it... I hoped it would help with the xml styling.